music-player/src/components/SongsComponent.vue

```diff
@@ -2,6 +2,15 @@
   <p>
     Current playlist: <strong>{{ selectedPlaylist.name }}</strong>
   </p>
+
+  <ol v-if="selectedPlaylist.SongsList.length !== 0">
+    <li v-for="song in selectedPlaylist.SongsList" :key="song.title">
```

Comment:
We should use index as a key or We should add some indicator to the playlist to avoid probable corruption. Bear in mind that We can have playlists with the same title